pkg/rootless/rootless_linux.c

```diff
@@ -254,6 +254,15 @@ do_preexec_hooks_dir (const char *dir, char **argv, int argc)
 static void
 do_preexec_hooks (char **argv, int argc)
 {
+  FILE *preexec_ptr;
+
+  // Open the preexec_hooks_dir indicator file in read mode
+  // return without processing if the file doesn't exist
+  preexec_ptr = fopen("/etc/containers/podman_preexec_hooks.txt", "r");
```

Comment:
since we are checking only its existence, we can just use `access()`

Reply:
And more rust flaking, I'd forgotten about access, that works out much nicer, ty.